docpages/include/coro_warn.dox

```diff
@@ -0,0 +1 @@
+\warning D++ Coroutines are a very new feature and are currently only supported by D++ on g++ 11, clang/LLVM 14, and MSVC 19.37 or above. Additionally, D++ must be built with the CMake option DPP_CORO, and your program must both define the macro DPP_CORO and use C++20 or above. The feature is experimental and may have bugs or even crashes, please report any to <a href="https://github.com/brainboxdotcc/DPP/issues">GitHub Issues</a> or to our <a href="https://discord.gg/dpp">Discord Server</a>.
```

Comment:
why `<a href>text</a>` and not `[text](url)`? Is `[]()` not allowed here?

Reply:
No this doesn't work with \include, sadly

Reply:
Ahhh, that's silly, got ya :)